zivid_camera/test/test_zivid_camera.cpp

```diff
@@ -36,11 +50,14 @@ class ZividNodeTest : public testing::Test
   const ros::Duration dr_get_max_wait_duration{ 1 };
   static constexpr auto capture_service_name = "/zivid_camera/capture";
   static constexpr auto capture_2d_service_name = "/zivid_camera/capture_2d";
+  static constexpr auto capture_assistant_suggest_settings_service_name = "/zivid_camera/capture_assistant/"
+                                                                          "suggest_settings";
   static constexpr auto color_camera_info_topic_name = "/zivid_camera/color/camera_info";
   static constexpr auto color_image_color_topic_name = "/zivid_camera/color/image_color";
   static constexpr auto depth_camera_info_topic_name = "/zivid_camera/depth/camera_info";
   static constexpr auto depth_image_raw_topic_name = "/zivid_camera/depth/image_raw";
   static constexpr auto points_topic_name = "/zivid_camera/points";
+  static constexpr size_t num_dr_capture_servers = 10;
```

Comment:
I think all the above parameters could be set from the launch file. This way we could avoid syncing both here and in the samples if we change the wait duration, topic names or the number of dynamic reconfigure services.

I think we could keep it like this if we are certain that these parameters will not change (the user can use `remap` if they need another topic name for instance for the points). I am also open to doing this as a separate issue, as it would require some other changes, for instance in the samples.